packages/forma-36-website/src/pages/guidelines/date-and-time/index.mdx

```diff
@@ -0,0 +1,206 @@
+      <TableCell style={{ width: '50%' }}>Notes</TableCell>
+    </TableRow>
+  </TableHead>
+  <TableBody>
+    <TableRow>
+      <TableCell>Absolute</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>13 Aug 2019 at 10:00 am</TableCell>
+      <TableCell>
+        When is an entry scheduled to be published?
+        <br />
+        Relative time wouldn't be specific enough.
+      </TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Relative</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        <Tooltip content="13 Aug 2019 at 10:00 am">3 days ago</Tooltip>
+      </TableCell>
+      <TableCell>
+        When was my collaborator last active?
+        <br />
+        Absolute date and time would be unnecessarily specific. Provide the
+        absolute date and time on user interaction, e.g. when the user hovers
+        over relative time with their cursor.
+      </TableCell>
+    </TableRow>
+  </TableBody>
+</Table>
+
+## Formats
+
+<Table className="f36-margin-bottom--xl" style={{ width: '100%' }}>
+  <TableHead>
+    <TableRow>
+      <TableCell style={{ width: '25%' }}>Type</TableCell>
+      <TableCell style={{ width: '25%' }}>Example</TableCell>
+      <TableCell style={{ width: '50%' }}>Notes</TableCell>
+    </TableRow>
+  </TableHead>
+  <TableBody>
+    <TableRow>
+      <TableCell>Relative date and time</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        Never
+        <br />
+        Less than an hour ago
+        <br />
+        13 days ago
+        <br />
+        3 months ago
+        <br />a year ago
+      </TableCell>
+      <TableCell></TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Absolute time</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>8:00 am</TableCell>
+      <TableCell>
+        Time should be displayed in 12-hour format, with lowercase am or pm and
+        without periods, to increase scannability.
+      </TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Absolute dates</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>13 Jan 2019</TableCell>
+      <TableCell>
+        Dates should be displayed in order of day-month-year, with month
+        abbreviated and without punctuation, to increase scannability and to
+        strike a compromise between different localized date formats.
+      </TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Absolute date and time</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>8 Aug 2019 at 8:00 am</TableCell>
+      <TableCell>
+        Absolute date and time should be displayed without commas and as
+        human-readable as possible.
+      </TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Weekday</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        Mon, 12 Aug
+        <br />
+        Wed, 14 Aug at 11:00 pm
+      </TableCell>
+      <TableCell>
+        Weekdays should be used when users need to compare events that happen in
```

Comment:
When a user needs to compare events that happen in daily context, weekdays should be used. Display weekdays next to absolute date and time, separated with a comma.

Reply:
This sentence sounds strange in general. I'm gonna write it again 👍